test/e2e/info_test.go

```diff
@@ -152,4 +152,19 @@ var _ = Describe("Podman Info", func() {
 		Expect(session.OutputToString()).To(ContainSubstring("memory"))
 		Expect(session.OutputToString()).To(ContainSubstring("pids"))
 	})
+
+	It("Podman info: check desired runtime", func() {
+		// defined in .cirrus.yml
+		want := os.Getenv("CI_DESIRED_RUNTIME")
+		if want == "" {
+			if os.Getenv("CIRRUS_CI") == "" {
+				Skip("CI_DESIRED_RUNTIME is not set--this is OK because we're not running under Cirrus")
```

Comment:
Would it make sense to just define `CI_DESIRED_RUNTIME` in the Makefile?  That could avoid all this checking-complexity.  CI always runs through the Makefile, and any developer running locally should be doing so as well.

Reply:
Absolutely not. The Makefile has no knowledge of which VMs want crun and which want runc. Only `.cirrus.yml` should know that.

Although these tests add some duplication, the reason is that I never, ever, _ever_ want to go through this nightmare week again. Nor do I want anyone else to.

Reply:
Relating to `get_ci_vm.sh`: `$CIRRUS_CI` will be unset, but `$CI_DESIRED_RUNTIME`  will be set.

In any case, I'm confused by the mismatch between the condition and skip-message.  In other words, the message mentions one var. but the actual condition is for a different var.

Reply:
```
if $CI_DESIRED_RUNTIME is not set
    if $CIRRUS_CI is not set
        skip, clearly indicating that this is OK
```